code/ship/ship.cpp

```diff
@@ -1768,77 +1768,112 @@ ship_info::~ship_info()
 	free_strings();
 }
 
+static SCP_vector<SCP_string> Removed_ships;
```

Comment:
Removed_ships is only every pushed into, and never removed from.
On first sight, that looks like a memory problem.
Maybe add a comment, to explain how the size of the vector is limited (if it is).

Reply:
I'll add a comment.

Reply:
Or something else that I think makes more sense.